src/freenet/node/Node.java

```diff
@@ -2027,21 +2028,20 @@ public void set(Boolean val) throws InvalidConfigValueException,
 				"Node.storeSaltHashSlotFilterPersistenceTime", "Node.storeSaltHashSlotFilterPersistenceTimeLong", new IntCallback() {
 
 					@Override
-					public Integer get() {
-						return ResizablePersistentIntBuffer.getPersistenceTime();
-					}
+					public Integer get() { return storeSaltHashSlotFilterPersistenceTime;}
```

Comment:
Please don’t collapse this into a single line; otherwise this looks good.

Reply:
Done! Yeah that looked weird!